### DISCUSSION TITLE
2023-02-16のミーティングアジェンダ

### DISCUSSION THREAD

**Discussion by azu** [asciidwango/js-primer#1589] [Meeting]
2023-02-16のミーティング置き場

前回: https://github.com/asciidwango/js-primer/discussions/1544

**Comment by azu** [comment]
Node/npmのアップデート

- [refactor: Update to Node.js 18.14.0 and npm 9.3.1 by azu · Pull Request #1567 · asciidwango/js-primer](https://github.com/asciidwango/js-primer/pull/1567)

**Reply by azu** [reply]
[NPM 9 update in 18.14.0 broke `npm ci` with local dependencies · Issue #46542 · nodejs/node](https://github.com/nodejs/node/issues/46542)

**Comment by azu** [comment]
2月末

- [書籍版へ差分を反映 · Issue #1591 · ascii
dwango/js-primer](https://github.com/asciidwango/js-primer/issues/1591)
- [第二版のレビューアーを追加する · Issue #1590 · asciidwango/js-primer](https://github.com/asciidwango/js-primer/issues/1590)

**Reply by lacolaco** [reply]
そういえば teamのdiscussion機能をクローズするとかGitHub から来てた気がする 今後レビュアー限定のやりとりに使えなくなる

> Team Discussions will be deprecated on May 8, 2023 in favor of GitHub Discussions. https://docs.github.com/en/organizations/collaborating-with-your-team/about-team-discussions

**Reply by azu** [reply]
Markdownから表記揺れる

**Reply by azu** [reply]
https://github.com/asciidwango/js-primer/blob/master/tools/strip-markdown-extension.cjs

**Reply by azu** [reply]
3/6 から1週間ぐらいPDFのレビューをする

**Reply by azu** [reply]
6月頭目標

**Reply by azu** [reply]
2月末に一旦全部入れてフリーズする

**Comment by azu** [comment]
- [x] Open Collectiveのアプリを許可してもう

**Comment by azu** [comment]
https://github.com/tc39/proposals/blob/main/finished-proposals.md

**Reply by azu** [reply]
https://github.com/tc39/proposal-hashbang

**Reply by azu** [reply]
コメントは3種類から4種類になる。

**Comment by azu** [comment]
次回 2023/03/13 19:00
